charts/helm-controller-stack/README.md

```diff
@@ -0,0 +1,103 @@
+# helm-controller-stack
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.16.0](https://img.shields.io/badge/AppVersion-1.16.0-informational?style=flat-square)
+
+A Helm controller operator chart for Kubernetes
+
+Have a look on official documentation [Using Helm Operator in Kubernetes clusters](https://github.com/fluxcd/helm-controller)
```

Comment:
The description of this chart is quite minimal, a short paragraph explaining what it is, with a link to the flux repository might be relevant here.